index.html

```diff
@@ -51,107 +51,59 @@ <h1 class="mainHeader">
-                    </p>
-                </div>
+            <div class="mainBox">
+                <img src="images/workshop.png" alt="workshop logo" width="40px" height="40px"/>
+                <h2>Workshop</h2>
+                <p>Helping thousands of members to start their own work.</p>
             </div>
-            <div class="card">
-                <div class="img4"></div>
-                <div class="description">
-                    <h3 class="name">Alfie Yang</h3>
-                    <small class="italic">Web designer of Techscape@3AM</small>
-                    <span class="grayBar"></span>
-                    <p class="text">
-                        Alfie is a passionate web designer with a great sense of style, sharing his perspectives minimalistic designs and perfectionalism.
-                    </p>
-                </div>
+            <div class="mainBox">
+                <img src="images/network.png" alt="Network logo" width="40px" height="40px"/>
+                <h2>Network</h2>
+                <p>Connect with professionals within a growing community.</p>
             </div>
-            <div class="card">
-                <div class="img5"></div>
-                <div class="description">
-                    <h3 class="name">Phillip Paing</h3>
-                    <small class="italic">CEO of Pareon and expertise in digital marketing</small>
-                    <span class="grayBar"></span>
-                    <p class="text">
-                        Paing has always dreamed about being an enterprenuer, starting his journey from UCONN and founded Pareon in 2022.
-                    </p>
-                </div>
+        </div>
+        <button id="join" type="submit">Join Us</button>
+        <a id="seeMore" href="#">See the whole program</a>
+    </main>
+    <section class="speakers"></section>
+    <section class="partner">
+        <h4>Partner</h4>
+        <span class="partnerBar"></span>
+        <div class="partnerContainer">
+            <div class="firstRow">
+                <p>mozilla</p>
+                <p>Google</p>
+                <p>NAVER</p>
             </div>
-            <div class="card">
-                <div class="img6"></div>
-                <div class="description">
-                    <h3 class="name">Ko PI</h3>
-                    <small class="italic">CEO of Techscape@3AM</small>
-                    <span class="grayBar"></span>
-                    <p class="text">
-                        Ko PI established the startup in 2017 with his fellows. Sharing technological information to people with blogs, vlogs and conferences.
-                    </p>
-                </div>
+            <div class="secondRow">
+                <p>daumkakao</p>
+                <p class="airbnb"><i class="fa-brands fa-airbnb"></i>airbnb</p>
             </div>
         </div>
     </section>
```

Comment:
- [ ] Like the About first and second sections, there is an overlap happening on the mobile view of the partners and speakers section of the Home page. Kindly fix the styling of this section so there is no overlap in any screen size. Be aware of the use of `vh` or `vw` values when sizing elements, screen sizes can be different and it is hard to make it responsive for all screen sizes.
 - Your design:
 - 
<img width="488" alt="image" src="https://user-images.githubusercontent.com/61919617/225437044-94375904-0cee-4709-9d43-5000653b07c3.png">

Reply:
:+1:

Reply:
## :+1: